.drone.star

```diff
@@ -1169,6 +1301,59 @@ def getDbDatabase(db):
 
 	return 'owncloud'
 
+def cacheRestore():
+	return [{
+		'name': 'cache-restore',
+		'image': 'plugins/s3-cache:1',
+		'pull': 'always',
+		'settings': {
+			'access_key': {
+				'from_secret': 'cache_s3_access_key'
+			},
+			'endpoint': {
+				'from_secret': 'cache_s3_endpoint'
+			},
+			'restore': True,
+			'secret_key': {
+				'from_secret': 'cache_s3_secret_key'
+			}
+		},
+		'when': {
+			'instance': [
+				'drone.owncloud.services',
+				'drone.owncloud.com'
+			],
+		}
+	}]
+
+def composerInstall(phpVersion):
+	return [{
+		'name': 'composer-install',
+		'image': 'owncloudci/php:%s' % phpVersion,
+		'pull': 'always',
+		'environment': {
+			'COMPOSER_HOME': '/drone/src/.cache/composer'
+		},
+		'commands': [
+			'make install-composer-deps'
```

Comment:
```suggestion
			'make vendor'
```

The apps all have a `vendor` target in their Makefile, and that does `composer install` to install the PHP dependencies. So that should work here.

CI crashed https://drone.owncloud.com/owncloud/data_exporter/1398/22/3
make: *** No rule to make target 'install-composer-deps'.  Stop.

Reply:
done